linux_os/guide/system/accounts/accounts-session/accounts_max_concurrent_login_sessions/ansible/shared.yml

```diff
@@ -3,13 +3,36 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-- (xccdf-var var_accounts_max_concurrent_login_sessions)
+
+{{{ ansible_instantiate_variables("var_accounts_max_concurrent_login_sessions") }}}
+
+- name: Find /etc/security/limits.d files contains maxlogins configuration
+  shell: "grep -q '^[^#]*\\<maxlogins\\>' /etc/security/limits.d/*.conf"
+  register: maxlogins
+  failed_when: False
```

Comment:
Maybe a `find` task with `contains` and `patterns` would work here?
This macro could be inspiring: https://github.com/ComplianceAsCode/content/blob/master/shared/macros-ansible.jinja#L359

Reply:
OMG, I overlooked `contains` option in `find` module. At first, I thought `patterns` was what I wanted, but of course it didn't work. Then I gave up and implemented with `shell`. I'll try `find` again and see what happens. Thank you.